scripts/generate_conda_file.sh

```diff
@@ -90,14 +90,12 @@ ${pyspark}- pyspark==2.3.1
 - fastparquet>=0.1.6
 ${pyspark}- pyarrow>=0.8.0
 - pip:
-  - idna==2.7
```

Comment:
@yueguoguo  @loomlike I think we are not using idna or tffm at the moment, am I right?

Reply:
`idna` is a dependency package required by some other packages. Here, the explicit addition of `idna` with a fixed version of 2.7 is because there are conflicts of package versions for certain packages. The consequence of having no fixed version number of `idna` is that it would produce errors in setting up the conda environment in the CI/CD machine. So let's keep it.

Reply:
good, I put the library back